Test/SilacTest.cs

```diff
@@ -409,13 +409,13 @@ public static void TestSilacTurnover()
 
             output = File.ReadAllLines(TestContext.CurrentContext.TestDirectory + @"/TestSilac/AllQuantifiedPeaks.tsv");
             Assert.AreEqual(output.Length, 4); //header, unlabeled, mixed, labeled
-            Assert.IsTrue(output[1].Contains("\tPEPTK(+8.014)IDEK(+8.014)\t")); //test the original is present
-            Assert.IsTrue(output[1].Contains("\t218750\t")); //test intensity
-            Assert.IsTrue(output[2].Contains("\tPEPTK(+1.994)IDEK(+8.014)\t")); //test human readable label (and lack thereof) is present
-            Assert.IsTrue(output[2].Contains("\t875000\t")); //test intensity
-            Assert.IsTrue(output[3].Contains("\tPEPTK(+1.994)IDEK(+1.994)\t")); //test other label is present
-            Assert.IsTrue(output[3].Contains("\t437500\t")); //test intensity
-            Assert.IsTrue(output[3].Contains("silac\t")); //test human readable labels are present
+            Assert.IsTrue(output[3].Contains("\tPEPTK(+8.014)IDEK(+8.014)\t")); //test the original is present
```

Comment:
The new FlashLFQ (in mzLib) version writes non-zero intensity peaks first, so the ordering of the peaks in the written file is different than what this test expected and the unit test had to be edited a bit to expect a different order.